CMakeLists.txt

```diff
@@ -67,14 +68,23 @@ if(BUILD_TOOLS)
         set_target_properties(gltfpack PROPERTIES INSTALL_RPATH ${RPATH})
     endif()
 
-    if(EXISTS glfw)
+    find_package(glfw3 3.3 QUIET)
+
+    if (glfw3_FOUND)
+        message(STATUS "Found GLFW3 via find_package()")
+        add_executable(lodviewer tools/lodviewer.cpp tools/meshloader.cpp)
+        target_link_libraries(lodviewer ${GLFW3_LIBRARY} meshoptimizer)
+    elseif(EXISTS ${BUILD_TOOLS_GLFW_FOLDER_NAME})
```

Comment:
This should probably be renamed to smth like MESHOPT_GLFW_PATH

Reply:
Also, would it be possible to use a defined check vs exists? I'd like it to be the case that if the option is specified, we *always* act on it - letting CMake produce errors if the path is incorrect.